releasenotes/index.md

```diff
@@ -6,6 +6,7 @@ permalink: /releasenotes/
 
 The Release Notes and Javadocs for every major release of Apache jclouds are listed chronologically below.
 
+* [1.9.2](/releasenotes/1.9.2) | [Javadoc](/reference/javadoc/1.9.x/)
```

Comment:
Have we published the 1.9.2 Javadoc yet? It looks like we might have even forgotten to publish 1.9.1 ;-)

http://jclouds.apache.org/reference/javadoc/1.9.x/